Foreman/Mod.cs

```diff
@@ -1,21 +1,20 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Text;
 
 namespace Foreman
 {
 	public class Mod
 	{
-        public String Id = "";
-		public String Name = "";
-		public String title = "";
-		public String version = "";
+		public string Id = "";

```

Comment:
I'm no expert in C# so I'm probably just ignorant but what's the purpose behind changing these to "string"?

Reply:
`string` is just an alias for `System.String` just like `int` is an alias to `System.Int32`.
It doesn't change anything, I'm just used to string and most code in C# use string for the type and String for the methods.
I will revert back to use String here though for consistency in the project.